### ISSUE TITLE
Bot is not responding to some commands

### ISSUE THREAD

**Issue by thiagonaalmeida** [group-butler/GroupButler#373]
After the server problem was turned off on day 11, the bot is experiencing some problems, I have stopped and restarted it, but it still has problems.

I am trying to edit the group rules, but the /setrules and /setrules - commands are not being answered.

The /config command also does not open the configuration options keyboard.

**Comment by yangm97** [comment]
Does the issue persist? I made some changes.

**Comment by thiagonaalmeida** [comment]
Yes, the problem persists, I just tested several commands and none of them responded.

**Comment by yangm97** [comment]
@thiagonaalmeida can you reproduce the issue on @GroupButler_bot or does it happen only on your clone?

**Comment by thiagonaalmeida** [comment]
Yes, the issue is in the @GroupButler_bot, I do not have a clone, I use the bot in my group.

I try again just a few minutes ago, and the problem persists.

**Comment by yangm97** [comment]
<img width="492" alt="screen shot 2018-10-02 at 16 59 05" src="https://user-images.githubusercontent.com/4412002/46373463-7a8eac80-c664-11e8-8a89-4181e962e804.png">

It's working fine for me, are you sure you don't have broken markdown?

**Comment by thiagonaalmeida** [comment]
No, I don't have a broken markdown. The issues started after the problem with the server. After this, the bot stop responding my commands.

**Comment by thiagonaalmeida** [comment]
Hi Yan,

my problems still persists.
The bot not respond various commands, I try to stop and restart many times, I can't modify the rules or welcome msgs. 
Is there a email to send some screens?
